frontend/packages/core/src/card.tsx

```diff
@@ -126,20 +119,90 @@ const BaseCardContent = styled.div<SpacingProps>`
   ${spacing}
 `;
 
-const StyledCardContent = styled(BaseCardContent)({
+const StyledCardContentContainer = styled.div({
   "> .MuiPaper-root": {
     border: "0",
     borderRadius: "0",
   },
+  overflow: "hidden",
 });
 
+const BaseCardActionArea = styled(CardActionArea)<SpacingProps>`
+  ${spacing}
+`;
```

Comment:
similar to `CardContent`, spacing on the `CardActionArea` should be configurable and we resuse this component for the landing card